### ISSUE TITLE
Error While Trying to Gather All Messages from a Group

### ISSUE THREAD

**Issue by dvmorris** [rhgrant10/Groupy#23]
I tried to follow the tutorial that uses messages.iolder() within a while loop, and I get the following error. Any idea what I might be doing wrong?

```
>>> while messages.iolder():
...     pass
...
Traceback (most recent call last):
  File "/Users/dvmorris/Documents/Groupy/groupy/api/endpoint.py", line 69, in response
    data = r.json()
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/site-packages/requests/models.py", line 808, in json
    return complexjson.loads(self.text, **kwargs)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/json/__init__.py", line 319, in loads
    return _default_decoder.decode(s)
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/Library/Frameworks/Python.framework/Versions/3.5/lib/python3.5/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/Users/dvmorris/Documents/Groupy/groupy/object/responses.py", line 126, in messages
    since_id=since, after_id=after)
  File "/Users/dvmorris/Documents/Groupy/groupy/api/endpoint.py", line 303, in index
    return cls.response(r)
  File "/Users/dvmorris/Documents/Groupy/groupy/api/endpoint.py", line 71, in response
    raise errors.ApiError(r)
groupy.api.errors.ApiError: <Response [304]>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/Users/dvmorris/Documents/Groupy/groupy/object/listers.py", line 168, in iolder
    old = self.older()
  File "/Users/dvmorris/Documents/Groupy/groupy/object/listers.py", line 145, in older
    return self.group.messages(before=self.oldest.id)
  File "/Users/dvmorris/Documents/Groupy/groupy/object/responses.py", line 129, in messages
    if e.args[0]['code'] == status.NOT_MODIFIED:
TypeError: 'Response' object is not subscriptable
```

**Comment by rhgrant10** [comment]
Well I'm mobile at the moment, but it looks like I was expecting the
exception argument to be a dictionary of JSON returned from the API in the
case of a bad request. However, it appears there is no JSON being returned
at all, which leads to the first exception, and ultimately leads to the
last exception (which occurs during the handling of the first exception).

I'll try to release a fix for this over the weekend as I'm rather bogged
down with work this week. Thanks for bringing this to my attention and I'll
keep you abreast of the situation.

**Comment by dvmorris** [comment]
No problem at all. Thanks for getting back to me so quickly! Let me know if I can help in any way.

**Comment by dvmorris** [comment]
Were you able to look into this issue?

**Comment by rhgrant10** [comment]
I'm working to fix it this morning. Hopefully it goes smoothly :)

**Comment by rhgrant10** [comment]
@dvmorris @mmirate This fix should be available on the PyPI shortly as version 0.6.6! Thanks for pointing out this issue :)

I'm currently experiencing a 500 error trying to upload it. I expect they're having some issues at the moment, but I'll keep trying throughout the day.

**Comment by dvmorris** [comment]
Thanks! The fix worked for me!
